page/login.jsx

```diff
@@ -14,12 +14,45 @@ css(`
 }
 `)
 
-
 export const Login = () => (
   <Div class="div-login">
-    <P fg="comment">{logo.split("#")}</P>
-    <NavLink href={`${API}/link/github`}  style={{fontSize: 30}}>
-      Join with Github
+    <P fg="comment">{logo.split('#')}</P>
+
+    <P fg="comment">
+      {'\n'}
+      <strong>NB : </strong>
+      you should have a github and discord account first.{'\n'}
+      If you don't have one, please create them before connecting.{'\n'} create
+      a github account{' '}
+      <NavLink
+        style={{ cursor: 'pointer' }}
+        target="_blank"
+        href="https://www.github.com"
+      >
+        https://www.github.com
+      </NavLink>
+      {'\n'} create a discord account{' '}
+      <NavLink
+        style={{ cursor: 'pointer' }}
+        target="_blank"
+        href="https://www.discord.com"
+      >
+        https://www.discord.com
+      </NavLink>
+    </P>
+    <br />
+    <NavLink href={`${API}/link/github`} style={{ fontSize: 30 }}>
+      <button
+        style={{
+          outline: 'none',
+          background: '#6070a4',
+          padding: '0.5rem',
+          borderRadius: '12px',
+          cursor: 'pointer',
+        }}
+      >
+        Join with Github
+      </button>
     </NavLink>
   </Div>
```

Comment:
we already have a nice way to apply colors to components using pre-defined colors, here is the full list:
```js
[
  "background-darker",
  "background-dark",
  "background",
  "background-light",
  "background-lighter",
  "selection-darker",
  "selection-dark",
  "selection",
  "selection-light",
  "selection-lighter",
  "foreground-darker",
  "foreground-dark",
  "foreground",
  "foreground-light",
  "foreground-lighter",
  "comment-darker",
  "comment-dark",
  "comment",
  "comment-light",
  "comment-lighter",
  "cyan-darker",
  "cyan-dark",
  "cyan",
  "cyan-light",
  "cyan-lighter",
  "green-darker",
  "green-dark",
  "green",
  "green-light",
  "green-lighter",
  "orange-darker",
  "orange-dark",
  "orange",
  "orange-light",
  "orange-lighter",
  "pink-darker",
  "pink-dark",
  "pink",
  "pink-light",
  "pink-lighter",
  "purple-darker",
  "purple-dark",
  "purple",
  "purple-light",
  "purple-lighter",
  "red-darker",
  "red-dark",
  "red",
  "red-light",
  "red-lighter",
  "yellow-darker",
  "yellow-dark",
  "yellow",
  "yellow-light",
  "yellow-lighter",
  "grey-light",
 "grey-lighter",
"grey",
"white",
"white-lighter",
"white-light"
]
```
all available through the class token properties bg and fg or css variables var(--yellow)
use this list of color instead of code color